MaterialGraphProject/Assets/UnityShaderEditor/Editor/Data/Implementation/NodeUtils.cs

```diff
@@ -207,5 +208,17 @@ public static string GetHLSLSafeName(string input)
             arr = Array.FindAll<char>(arr, (c => (Char.IsLetterOrDigit(c))));
             return new string(arr);
         }
+
+        public static string FloatToShaderValue(float value)
```

Comment:
Can we add something to handle E notation here? This is causing errors also.

Reply:
Ok I have done that.

Reply:
LGTM!